requirements-dev.txt

```diff
@@ -1,5 +1,4 @@
-pre-commit==3.1.1; python_version >= '3.8'
-pre-commit==2.21.0; python_version < '3.8'
 codecov==2.1.12
+pre-commit==2.21.0; python_version < '3.8'
+pre-commit==3.1.1; python_version >= '3.8'
```

Comment:
Already updated in https://github.com/python-social-auth/social-app-django/pull/434 so I think you can drop this change.